core/src/test/java/org/apache/calcite/plan/RelOptUtilTest.java

```diff
@@ -45,6 +47,8 @@
 import org.apache.calcite.util.TestUtil;
 import org.apache.calcite.util.Util;
 
+import org.apache.commons.lang3.tuple.Pair;
```

Comment:
Calcite has an implementation of Pair. Just curious:

is it preferred to use Calcite's Pair. If the answer is no then we can mark Calcite's `Pair` as `@derepcated` to help new code migrated to other Pair implementation.

Reply:
I did a quick codebase search and found no usage of `org.apache.commons.lang3.tuple.Pair`.

Reply:
Yes, thanks.